### PR TITLE
Fixed PropTypes and React.createClass deprecation warnings

### PR DESCRIPTION
**Pull request by pista329**
Basically solution for this stuck PR https://github.com/JedWatson/react-hammerjs/pull/67

### CODE REVIEW

**Comment by pista329** [comment]
Can anyone pls review this and possibly accept and push to npm?

**Comment by rdbmax** [comment]
@JedWatson :)

**Comment by renatorib** [comment]
@JedWatson I think this is a high priority, can you see it? Only merge && publish

**Comment by evan-scott-zocdoc** [comment]
@JedWatson could you please cut this? Happy to help with the release if you want.

**Comment by rhys-vdw** [comment]
@JedWatson bump! (please 😄 )

**Comment by jrsinclair** [comment]
This just became slightly more pressing, now that React 16 has been released.

**Comment by Redmega** [comment]
We are using hammer and are eager to update to react 16! Any movement on this?

File: package.json [hunk 1]
```diff
@@ -25,6 +26,7 @@
     "gulp-uglify": "^1.5.4",
     "gulp-util": "^3.0.7",
     "merge-stream": "^1.0.0",
+    "prop-types": "^15.5.10",
```
Comment (lostthetrail) [inline package.json]: - [ ] [Move this to be a dependency](https://github.com/facebook/prop-types#how-to-depend-on-this-package)

File: package.json [hunk 2]
```diff
@@ -34,7 +36,9 @@
   },
```
Comment (lostthetrail) [inline package.json]: - [ ] `peerDependencies.react` should be updated to include `|| ^16.0.0`.
- [ ] `peerDependencies` should include `"react-dom": "^0.14.3 || ^15.0.0 || ^16.0.0"`

**Comment by pista329** [comment]
No longer relevant.

**Comment by natenorberg** [comment]
Why is this no longer relevant? Is there a different PR that adds React 16 support? Or does this just mean go find a different component?

**Comment by natenorberg** [comment]
Got it. Thanks!

**Pull request closed without merging**